packages/components/src/loader/src/Loader.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState, useEffect, ComponentProps, forwardRef } from "react";
+import { InternalProps, OmitInternalProps, StyledComponentProps, cssModule, mergeProps, isNil, isNumber } from "../../shared";
+import { Box } from "../../box";
+import { Div } from "../../html";
+
+const DefaultElement = "div";
+
+const DefaultLoaderDelay = 300;
+
+export interface InnerLoaderProps extends InternalProps, Omit<StyledComponentProps<typeof DefaultElement>, "children"> {
+    /**
+     * Specify whether you want the Loader to be animating or not.
+     */
+    active?: boolean;
+    /**
+     * See [WCAG](https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html).
+     */
+    "aria-label": string;
+    /**
+     * A loader can appear after a short delay (in ms).
+     * if set the true, the default delay is 300ms.
+     */
+    delay?: boolean | number;
+}
+
+export function InnerLoader({
+    as = DefaultElement,
+    active = true,
+    delay,
+    forwardedRef,
+    ...rest
+}: InnerLoaderProps) {
+    const [show, setShow] = useState<boolean>(isNil(delay) || delay === false);
```

Comment:
I don't think it should be named "show" as it's not a prop of the components API. It should be something like "isVisible" or even "isShow"